src/components/ContextualMenu/ContextualMenuDropdown/ContextualMenuDropdown.tsx

```diff
@@ -224,7 +224,7 @@ const ContextualMenuDropdown = <L,>({
         }
       >
         {dropdownContent
-          ? dropdownContent
+          ? React.cloneElement(dropdownContent as any, { close: closePortal })
```

Comment:
This will cause issues for any component that does not handle the `close` prop as it will try and pass it to the DOM (see the test error here: https://github.com/canonical/react-components/actions/runs/5221836743/jobs/9426601147?pr=928#step:7:3735).

Maybe this prop should accept either a `ReactNode` or a function that accepts a close prop and returns a `ReactNode`. That way it would be backwards compatible and also we'd get type checking of the render function.